scalyr_agent/monitor_utils/k8s.py

```diff
@@ -892,9 +900,11 @@ def _get_controller_from_owners(self, k8s, owners, namespace, query_options=None
 
             if controller.parent_kind not in _OBJECT_ENDPOINTS:
                 global_log.log(
-                    scalyr_logging.DEBUG_LEVEL_1,
+                    WARNING,
```

Comment:
Generally should use logging.WARNING instead, up to you if you want to change it

Reply:
that is what I'm using, I'm importing WARNING from logging up at the top.